ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -109,7 +109,7 @@ param set MAV_SYS_ID $((px4_instance+1))
 simulator_tcp_port=$((4560+px4_instance))
 udp_offboard_port_local=$((14580+px4_instance))
 udp_offboard_port_remote=$((14540+px4_instance))
```

Comment:
This will also overlap with the GCS port 14550.

Reply:
@bkueng Right. I was trying to do this without changing the offboard port since a lot of users rely on this. Would skipping just 14550 be a solution for this?

Reply:
What about using 14540-14549 for the first 10 instances, and then use 14549 for the rest? Assuming MAVSDK can handle that.